src/promptflow/promptflow/_sdk/entities/_flow.py

```diff
@@ -190,11 +195,15 @@ def load(
                 data = load_yaml_string(flow_content)
                 content_hash = hash(flow_content)
             is_eager_flow = cls._is_eager_flow(data)
+            is_async_flow = cls._is_async_flow(kwargs)
```

Comment:
Directly check async_call param here instead using a function? Since async is now a capability of `Flow` object

Reply:
OK